packages/realm/src/app-services/SyncSession.ts

```diff
@@ -286,11 +286,22 @@ export class SyncSession {
   // TODO: Figure out a way to avoid passing a mocked app instance when constructing the User.
   get config(): SyncConfiguration {
     const user = new User(this.internal.user, mockApp);
-    const { partitionValue, flxSyncRequested, customHttpHeaders } = this.internal.config;
+    const { partitionValue, flxSyncRequested, customHttpHeaders, clientValidateSsl, sslTrustCertificatePath } =
```

Comment:
Does the `clientValidateSsl` and `sslTrustCertificatePath` have sane defaults when coming from core? (i.e. not just an empty string or something).

Reply:
This will be whatever the user has provided as their `validate` (binding: `clientValidateSsl`) and `certificatePath` (binding: `sslTrustCertificatePath`).

```ts
export type SSLConfiguration = {
  validate?: boolean;
  certificatePath?: string;

  // ...
};
```

If you don't provide a value for `certificatePath`, the value can be undefined as in [this test](https://github.com/realm/realm-js/blob/244d77ddac0abf55a4a53691d8c184e6940b6677/integration-tests/tests/src/node/ssl.ts#L99). `validate` defaults to `true`.